litex/soc/interconnect/wishbonebridge.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, phy, clk_freq):
         ]
 
         fsm = ResetInserter()(FSM(reset_state="IDLE"))
-        timer = WaitTimer(clk_freq//10)
+        timer = WaitTimer(int(clk_freq//10))
```

Comment:
`clk_freq` is supposed to already be an` int` in the SoC.

Reply:
Probably so, but for the case that someone accidentally passes a float in (that would've been me), this avoids a hiccup and it has no downside as far as I can see.

Reply:
OK, we can add it yes, but using a non int `clk_freq` will probably case issues in others part the SoC. But we should do this to allow Modules in the future to avoid converting `sys_clk_freq` to int in the top level of the SoC.